content/nic/installation/ingress-nginx.md

```diff
@@ -488,6 +488,20 @@ nginx.com/sticky-cookie-services: "serviceName=example-svc cookie_name expires=t
 NGINX Ingress Controller has additional annotations for features using NGINX Plus that have no Ingress-NGINX Controller equivalent, such as active health checks and authentication using JSON Web Tokens (JWTs).
 {{< /call-out >}}
 
+The following table outlines annotation conversions for enabling TLS and gRPC protocols to backend (upstream) services.
+
+{{< bootstrap-table "table table-bordered table-striped table-responsive" >}}
```

Comment:
```suggestion
{{< table >}}
```

Reply:
why was this conversation marked as resolved? the issue has not been addressed